metricbeat/module/apache/status/data.go

```diff
@@ -11,239 +11,120 @@ import (
-			connsTotal, _ = strconv.Atoi(matches[1])
-		}
-
-		// ConnsAsyncWriting: 527
-		re = regexp.MustCompile("ConnsAsyncWriting: (\\d+)")
-		if matches := re.FindStringSubmatch(scanner.Text()); matches != nil {
-			connsAsyncWriting, _ = strconv.Atoi(matches[1])
-		}
-
-		// ConnsAsyncKeepAlive: 1321
-		re = regexp.MustCompile("ConnsAsyncKeepAlive: (\\d+)")
-		if matches := re.FindStringSubmatch(scanner.Text()); matches != nil {
-			connsAsyncKeepAlive, _ = strconv.Atoi(matches[1])
-		}
-
-		// ConnsAsyncClosing: 2785
-		re = regexp.MustCompile("ConnsAsyncClosing: (\\d+)")
-		if matches := re.FindStringSubmatch(scanner.Text()); matches != nil {
-			connsAsyncClosing, _ = strconv.Atoi(matches[1])
-		}
-
-		// ServerUptimeSeconds: 43
-		re = regexp.MustCompile("ServerUptimeSeconds: (\\d+)")
-		if matches := re.FindStringSubmatch(scanner.Text()); matches != nil {
-			serverUptimeSeconds, _ = strconv.Atoi(matches[1])
-		}
-
-		//Load1: 0.01
-		re = regexp.MustCompile("Load1: (\\d*.*\\d+)")
-		if matches := re.FindStringSubmatch(scanner.Text()); matches != nil {
-			load1 = parseMatchFloat(matches[1], hostname, "load1")
-		}
-
-		//Load5: 0.10
-		re = regexp.MustCompile("Load5: (\\d*.*\\d+)")
-		if matches := re.FindStringSubmatch(scanner.Text()); matches != nil {
-			load5 = parseMatchFloat(matches[1], hostname, "load5")
-		}
-
-		//Load15: 0.06
-		re = regexp.MustCompile("Load15: (\\d*.*\\d+)")
-		if matches := re.FindStringSubmatch(scanner.Text()); matches != nil {
-			load15 = parseMatchFloat(matches[1], hostname, "load15")
-		}
-
-		// Scoreboard Key:
-		// "_" Waiting for Connection, "S" Starting up, "R" Reading Request,
-		// "W" Sending Reply, "K" Keepalive (read), "D" DNS Lookup,
-		// "C" Closing connection, "L" Logging, "G" Gracefully finishing,
-		// "I" Idle cleanup of worker, "." Open slot with no current process
-		// Scoreboard: _W____........___...............................................................................................................................................................................................................................................
-		re = regexp.MustCompile("Scoreboard: (_|S|R|W|K|D|C|L|G|I|\\.)+")
-		if matches := re.FindStringSubmatch(scanner.Text()); matches != nil {
-			scr := strings.Split(scanner.Text(), " ")
+		} else {
 
-			totalUnderscore = strings.Count(scr[1], "_")
-			totalS = strings.Count(scr[1], "S")
-			totalR = strings.Count(scr[1], "R")
-			totalW = strings.Count(scr[1], "W")
-			totalK = strings.Count(scr[1], "K")
-			totalD = strings.Count(scr[1], "D")
-			totalC = strings.Count(scr[1], "C")
-			totalL = strings.Count(scr[1], "L")
-			totalG = strings.Count(scr[1], "G")
-			totalI = strings.Count(scr[1], "I")
-			totalDot = strings.Count(scr[1], ".")
-			totalAll = totalUnderscore + totalS + totalR + totalW + totalK + totalD + totalC + totalL + totalG + totalI + totalDot
+			logp.Warn("Unexpected line in apache server-status output: %s", scanner.Text())
```

Comment:
I would make this a Debug log message, as I don't think it should be a warning. The same can happen in Redis or MySQL and is more for debugging purpose. There can also be lines that were excluded on purpose.

This will also solve the "WARN" issue in the system tests.